examples/misc/bin/cheatsheet/nullable.dart

```diff
@@ -4,13 +4,13 @@ void main() {
   {
     // #docregion invalid-null
     // ignore: invalid_assignment
-    int a = null; // INVALID in null-safe Dart.
+    int a = null; // INVALID.
```

Comment:
To match the snippet in the file:

```suggestion
    int a = null; // INVALID
```